### PR TITLE
CHANGED: STYLE CHANGED, ADDED: BOOTSTRAP 4, JS

### PR DESCRIPTION
**Pull request by InclinedScorpio**
**Better User Interface, Bootstrap4 ADDED, JS Changed**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,22 +1,59 @@
 <!DOCTYPE html>
-<html>
+<ht
```
Comment (moderneinstein) [inline index.html]: can we please have the html tag in one line only

File: index.html [hunk 2]
```diff
@@ -1,22 +1,59 @@
...
-		Enter number of hours Absent:			<br><br>
-		<input type="number" name="absent" min="0">		<br><br>
-		<input type="submit" name="submit" value="Check">
+		Number of total hours Conducted	<br>
+		<input type="number" class="text-muted" name="conducted" min="1" placeholder="Check academia if not sure">	<br><br>
+		Number of hours Absent			<br>
+		<input type="number" class="text-muted" name="absent" min="0" placeholder="Check academia if not sure">		<br><br>
+		<input type="submit" name="submit" class="btn btn-lg btn-info" style="width:100%;" value="Check">
 	</form>
+	<center style="font-size:x-large;cursor: pointer;">
```
Comment (moderneinstein) [inline index.html:25]: can we please avoid using internal css and use 'style.css' instead?

File: index.html [hunk 3]
```diff
@@ -1,22 +1,59 @@
...
+
+
+	  </div>
+	  <hr>
+	</div> -->
+	<div class="container">
+	  <div class="row">
+		  <p class="copyright-text">Copyright &copy; 2020 All Rights Reserved by <a href="https://www.linkedin.com/in/harsh-srivastva/" target="_blank">Harsh</a> | <a href="https://www.linkedin.com/in/ashutosh3309/" target="_blank">Ashutosh</a>
+		  </p>
+
```
Comment (moderneinstein) [inline index.html]: can we please remove the extra blank lines here

File: index.html [hunk 4]
```diff
@@ -1,22 +1,59 @@
...
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
+
 	<title>Attendance</title>
 	<link rel="stylesheet" href="style.css">
 </head>
 <body>
 	<b><font size=6>Attendance Calculator</font></b>
-	<a href="https://academia.srmist.edu.in/#View:My_Attendance" target="_blank">
-		<button>SRM Academia</button>
+	<a class="a" href="https://academia.srmist.edu.in/#View:My_Attendance" target="_blank">
```
Comment (moderneinstein) [inline index.html]: can we have a better class name here

File: index.html (hunk 1 above)
Comment (InclinedScorpio) [reply index.html]: Implemented in eb31b00fe595c41cf526af9afff9453fb6aa9840

File: index.html (hunk 2 above)
Comment (InclinedScorpio) [reply index.html:25]: Implemented in eb31b00fe595c41cf526af9afff9453fb6aa9840

File: index.html (hunk 3 above)
Comment (InclinedScorpio) [reply index.html]: Implemented in eb31b00fe595c41cf526af9afff9453fb6aa9840

File: index.html (hunk 4 above)
Comment (InclinedScorpio) [reply index.html]: Implemented in eb31b00fe595c41cf526af9afff9453fb6aa9840

**Comment by InclinedScorpio** [comment]
Suggestions Implemented, Spaces, ClassNames

**Pull request merged**